src/main/java/org/alliancegenome/curation_api/constants/VocabularyConstants.java

```diff
@@ -63,4 +63,8 @@ private VocabularyConstants() {
 	public static final String GENE_GENETIC_INTERACTION_RELATION_VOCABULARY_TERM_SET = "gene_genetic_interaction_relation";
 	public static final String GENE_GENETIC_INTERACTION_RELATION_TERM = "genetically_interacts_with";
 	public static final String GENE_MOLECULAR_INTERACTION_RELATION_TERM = "physically_interacts_with";
-}
\ No newline at end of file
+
```

Comment:
Does this vocabulary and term already exist in the DB?  I don't see them in the migration.  Perhaps I missed them or they've been manually added to the curation system already?

Reply:
Manually created on alpha

Reply:
They also need to be manually created on beta and production then

Reply:
Migration would have been the safer route to avoid potential for errors/inconsistencies - might be worth deleting on alpha and adding to migration